### PR TITLE
LOOKDEVX-3144 - Fix transmission issues

### PR DESCRIPTION
**Pull request by JGamache-autodesk**
- Fix transparent detection on OpenPBR surface
- Fix transparent detection on glTf PBR
- Add OpenPBR to list of vetted materials

### CODE REVIEW

File: lib/mayaUsd/render/MaterialXGenOgsXml/OgsFragment.cpp [hunk 1]
```diff
@@ -423,6 +424,156 @@ std::string getOriginalName(const std::string& parameterName, const mx::NodeDefP
     return *it;
 }
 
+template <typename T>
+static bool
+parameterDiffersFrom(const mx::NodePtr& node, const std::string& paramName, T const& paramValue)
```
Comment (JGamache-autodesk) [inline lib/mayaUsd/render/MaterialXGenOgsXml/OgsFragment.cpp:429]: Copy of code introduced in USD in https://github.com/PixarAnimationStudios/OpenUSD/pull/3073 but adapted to MaterialX.

Please node associated MaterialX issue: https://github.com/AcademySoftwareFoundation/MaterialX/issues/1946

File: lib/mayaUsd/utils/util.cpp [hunk 2]
```diff
@@ -2542,10 +2542,12 @@ void UsdMayaUtil::AddMayaExtents(GfBBox3d& bbox, const UsdPrim& root, const UsdT
...
-    static const std::set<TfToken> vettedSurfaces = { TfToken("ND_standard_surface_surfaceshader"),
-                                                      TfToken("ND_gltf_pbr_surfaceshader"),
-                                                      TfToken("ND_UsdPreviewSurface_surfaceshader"),
-                                                      TfToken("UsdPreviewSurface") };
+    static const std::set<TfToken> vettedSurfaces
+        = { TfToken("ND_standard_surface_surfaceshader"),
+            TfToken("ND_gltf_pbr_surfaceshader"),
+            TfToken("ND_UsdPreviewSurface_surfaceshader"),
+            TfToken("UsdPreviewSurface"),
+            TfToken("ND_open_pbr_surface_surfaceshader") };
```
Comment (JGamache-autodesk) [inline lib/mayaUsd/utils/util.cpp:2550]: Adding it to the material creation menus if available.

**Review by frohnej-adsk — APPROVED** [review]
LGTM

**Review by vlasovi — APPROVED** [review]

**Comment by JGamache-autodesk** [comment]
@seando-adsk ready for merge.

**Pull request merged**